### PR TITLE
Bug 1127266 - Update test_rocketbar_search.py according to changes in Bu...

### PR DESCRIPTION
**Pull request by viorelaioia-zz**
...g 1117770

### CODE REVIEW

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e907c42cb3ca)

**Comment by viorelaioia** [comment]
http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/643/console

File: tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/search_panel.py [hunk 1]
```diff
@@ -93,10 +94,7 @@ def app_results(self):
 
     @property
     def link_results(self):
-        # A link result just opens a page in a frame
-        return [self.Result(marionette=self.marionette, element=result)
-                for result in self.marionette.find_elements(*self._search_results_locator)
-                    if not self._is_result_a_webapp(result)]
+        return self.marionette.find_elements(*self._link_results_locator)
```
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/search_panel.py]: why did we removed the `Result` object in this and we return only a list of webelements

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a4fb60c78489)

File: tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/search_panel.py (hunk 1 above)
Comment (viorelaioia) [inline tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/search_panel.py]: oh, this is a mistake, I'll undo it

**Comment by viorelaioia** [comment]
Adhoc run restarted: http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/646/console

**Pull request merged**